### DISCUSSION TITLE
Unable to execute scenario outline in parallel

### DISCUSSION THREAD

**Discussion by korrapatisandhya** [hrcorval/behavex#110] [Q&A] [answered]
i am using selenium, python ,behave and behavex libraries in my project.
My project is about to execute same steps for multiple input dynamically through command line parameters in that case i choose the scenario outline in behavex to use but i am unable execute the examples of scenario outline parallelly please let me know is behavex support parallel execution of scenario outline if so please provide an example and i attached code i am using.
![image](https://github.com/hrcorval/behavex/assets/130272434/14068bb9-63c3-4e97-9299-776c01d43542)
![image](https://github.com/hrcorval/behavex/assets/130272434/bd730ec0-0ed6-40ef-b4a8-55d228313841)

**Comment by hrcorval** [comment, marked as answer]
BehaveX was not supporting parallel execution of scenario outlines in previous versions, however, you can try version 3.2.0 that already supports that.
Hope it helps :)
